src/main/kotlin/com/hibob/academy/resource/PetsResource.kt

```diff
@@ -57,4 +57,22 @@ class PetsResource (private val petService: PetService){
         val resultMessage = petService.updatePetOwnerId(petId, newOwnerId, companyId)
         return Response.ok(resultMessage).build()
     }
+
+    @GET
+    @Path("getPetsByOwnerId/{ownerId}/companyId/{companyId}")
```

Comment:
redundant getPetsByOwnerId in path, but missing owner instead
you can read about rest naming conventions
https://restfulapi.net/resource-naming/